docs/pages/Guides/UpdatingCompose.md

```diff
@@ -0,0 +1,94 @@
+# Migrating from old to new compose
+
+The compose API off our components allows clients to supply new arguments for creating the control in order for them to meet their needs. In moving controls from the old to new framework, the compose API usage changes. The compose API has a large range of inputs and is very flexible. This guide will cover more common scenarios. If you need help with something not covered here, please open an issue on the repo.
```

Comment:
are [actual or stylistic] grammar comments desired through this CR?  It's nothing particularly wild, mostly style things; I think it's totally fine if our position is ```this file is temporary and not worth exceptional polish / feel free to open a PR to change it after this PR adds it; the priority for reviewing this PR is correctness``` 🙂

Example wordsmithing of the first paragraph https://github.com/PPatBoyd/fluentui-react-native/commit/5a64d3ab221ea1aa7a0a0bdf4bbf0616b3c606ea?short_path=6421069

Reply:
Thanks for the suggestions, I think your wording is better so took your suggestions